AUTHORS

```diff
@@ -549,3 +549,4 @@ Zakir Hussain Shaik <zakir.shaik@cockroachlabs.com> shaikzakiriitm <shaikzakir.i
 Zane Teh <zane@cockroachlabs.com>
 何羿宏 <heyihong.cn@gmail.com>
 智雅楠 <zac.zhiyanan@gmail.com>
+Rishabh Makhija <rishabh.makhija@cockroachlabs.com>
```

Comment:
Let's keep this file in alphabetical order. Can you move this to it's right location alphabetically?

Reply:
Done